tasks.go

```diff
@@ -13,8 +13,8 @@ type Task func() error
 
 // RunAllTasks runs all tasks known to the dump tool using concurrent workers.
 // Dump output goes to path.
-func RunAllTasks(path string, workers int) {
-	tasks := GetAllTasks(path)
+func RunAllTasks(runner Runner, path string, workers int) {
```

Comment:
why is `path string` required here, considering that the runner is constructed with it here: https://github.com/feedhenry/fh-system-dump-tool/pull/26/files#diff-7ddfb3e035b42cd70649cc33393fe32cR191